packages/core/lib/commands/debug/meta.js

```diff
@@ -35,6 +39,11 @@ module.exports = {
       OS.EOL +
       "                             [--compile-tests|--compile-all|--compile-none]",
     options: [
+      {
+        option: "--url",
+        description:
+          "Connects to a specified provider given via URL, ignoring networks in config. This option allows using the debugger outside of a Truffle project. That's why compileNone is set to true when a config file is not found AND url is specified"
```

Comment:
I think the the final sentence here needs to be cut.  That's implementation details that users don't need to know about and will just confuse them; also, it makes this description just too long.